deploy/helm/sumologic/README.md

```diff
@@ -142,6 +142,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `sumologic.collectionMonitoring`                                                      | If you set it to false, it would set `EXCLUDE_NAMESPACE=<release-namespace>` and not add the Prometheus remotestorage metrics.                                                                                                                                                                             | `true`                                                                                                                                                                               |
 | `sumologic.collectorName`                                                             | The name of the Sumo Logic collector that will be created in the SetUp job. Defaults to `clusterName` if not specified.                                                                                                                                                                                    | `Nil`                                                                                                                                                                                |
 | `sumologic.clusterName`                                                               | An identifier for the Kubernetes cluster. Whitespaces in the cluster name will be replaced with dashes.                                                                                                                                                                                                    | `kubernetes`                                                                                                                                                                         |
+| `sumologic.clusterDNSDomain`                                                          | The DNS domain for the cluster. Change this if you're using a custom domain.                                                                                                                                                                                                                               | `cluster.local`                                                                                                                                                                      |
```

Comment:
Do we need the `sumologic.` prefix to this property?

Reply:
It's there for consistency. Similar properties like the proxy settings live in this section as well.